src/impl_/wrap.rs

```diff
@@ -30,8 +30,9 @@ where
     T: IntoPy<PyObject>,
 {
     type Error = PyErr;
```

Comment:
I think the error here should be `Infallible`, as `PyErr` implements `From<Infallible>`

Reply:
Good idea 👍